terraform/environments/core-sandbox/playground.tf

```diff
@@ -19,16 +19,16 @@ module "ram-principal-association" {
 
 }
 
```

Comment:
As previously noted, this file doesn't need to be part of the commit. Can you remove the comments from the start of the lines here and raise an issue to fix this data call / take it on as a separate task?

Reply:
Will do

Reply:
Comments removed. Will raise an issue to fix this. Uses a very old version of linux so may no longer be available

Reply:
Removed the above from the PR so the original will stay in place.